src/app/clusters/scenes/SceneTableImpl.cpp

```diff
@@ -33,7 +33,8 @@ namespace scenes {
 /// kTransitionTime: Tag for the transition time of the scene in miliseconds
 enum class TagScene : uint8_t
 {
-    kSceneCount = 1,
+    kGlobalSceneCount = 1,
```

Comment:
Probably ok for now, but once we have these things stored we can't renumber tags like this....